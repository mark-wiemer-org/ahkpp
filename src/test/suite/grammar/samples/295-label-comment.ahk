; [Issue #295](https://github.com/mark-wiemer/vscode-autohotkey-plus-plus/issues/295)
OpenLabel: ;Comment
OpenLabel:;NotComment
