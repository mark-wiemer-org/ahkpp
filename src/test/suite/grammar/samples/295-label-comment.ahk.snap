>; [Issue #295](https://github.com/mark-wiemer/vscode-autohotkey-plus-plus/issues/295)
#^ source.ahk comment.line.semicolon.ahk punctuation.definition.comment.ahk
# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk comment.line.semicolon.ahk
>OpenLabel: ;Comment
#^^^^^^^^^^ source.ahk labelline.ahk entity.name.function.label.ahk
#          ^^^^^^^^^ source.ahk labelline.ahk comment.line.semicolon.label.ahk
>OpenLabel:;NotComment
#^^^^^^^^^ source.ahk variable.def.ahk
#         ^ source.ahk punctuation.ahk
#          ^ source.ahk
#           ^^^^^^^^^^ source.ahk variable.def.ahk
>